spec/models/incoming_mail_handler_spec.rb

```diff
@@ -1,9 +1,8 @@
 describe "Handles incoming email" do
```

Comment:
the changes in this file smell funny to me. did master really change so much here?